src/components/base/Dialogs/AlertDialog.tsx

```diff
@@ -26,15 +25,12 @@ const SwapConfirmation = (props: SwapConfirmationProps) => {
   const handleSwapTransaction = useCallback(async () => {
     const { tokenToTrade, tokenToReceive } = swapState;
     if (tokenToTrade.symbol === undefined || tokenToReceive.symbol === undefined) {
+      console.error("Token types must be selected to Swap tokens.");
       return;
     }
-    const tokenToTradeAccountId = TOKEN_SYMBOL_TO_ACCOUNT_ID.get(tokenToTrade.symbol) ?? "";
-    const tokenToReceiveAccountId = TOKEN_SYMBOL_TO_ACCOUNT_ID.get(tokenToReceive.symbol) ?? "";
     sendSwapTransaction({
-      depositTokenAccountId: tokenToTradeAccountId,
-      depositTokenAmount: tokenToTrade.amount,
-      receivingTokenAccountId: tokenToReceiveAccountId,
-      receivingTokenAmount: tokenToReceive.amount,
+      tokenToTrade: { ...tokenToTrade },
```

Comment:
Using spread to avoid passing references to old `swapState`. This shouldn't be an issue since `swapState` should be immutably updated in the Swap Reducer but I've decided on a defensive approach until tests are in place.